dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/metrics/PublishedFilesChart.java

```diff
@@ -87,16 +98,16 @@ private boolean isMonthlyChartSelected() {
     }
```

Comment:
Is it possible to use enum value also in ```isMonthlyChartSelected```?

Reply:
I'll go with `mode == ChartMode.MONTHLY && selectedYear != 0;` since it's important for `selectedYear != 0` when generating monthly chart.

Reply:
But then ```mode == ChartMode.MONTHLY``` would seem artificial, because the main selection logic depends on ``selectedYear != 0```. 
If there's no other way to do this, then let it stay.

Reply:
I guess we could check for `selectedYear != 0` somewhere on the chart generation process  and act accordingly. However checking it here, however not quite intuitive, is actually checking for scenario - monthly chart has been selected AND proper year has been choosen. So I'd rather leave it be as you say.